preloading/virustotal_download/virustotal_download.py

```diff
@@ -0,0 +1,71 @@
+from io import BytesIO
+
+from fame.core.module import PreloadingModule
+from fame.core.config import Config
+from fame.common.exceptions import (
+    ModuleExecutionError, ModuleInitializationError,
+    MissingConfiguration
+)
+
+try:
+    import requests
+    HAVE_REQUESTS = True
+except ImportError:
+    HAVE_REQUESTS = False
+
+
+class VirusTotalDownload(PreloadingModule):
+    name = "virustotal_download"
+    description = "Download file from VT and launch new analysis."
+    acts_on = ["hash"]
+
+    named_configs = {
+        'virustotal': {
+            'description': 'VirusTotal API configuration, in order to be able to submit hashes.',
+            'config': [
+                {
+                    'name': 'api_key',
+                    'description': 'VirusTotal Intelligence API key.',
+                    'type': 'str',
+                    'value': None
+                }
+            ]
+        }
+    }
```

Comment:
Since the VirusTotal download will be in a preloading module instead of in the core of FAME, it makes no more sense to use a `named_config`. It should probably be regular module configuration.

Reply:
The main reason to keep this as a named_config was simply to provide backwards compatibility. I, personally, agree that this could just be a module config - I leave that decision to you.

Reply:
The module would have to be manually enabled anyway, so I would rather use a regular module configuration.

Reply:
agreed and changed